gacl/docs/manual.html

```diff
@@ -9,7 +9,7 @@
 	<META NAME="CHANGED" CONTENT="20060903;16430300">
```

Comment:
I found this file has a lot of outdated and invalid code syntax, should I refactor it?

Reply:
Since this is a file from a third-party that isn't ours, we should leave this alone. I have an upcoming issue request for this sorta thing.

Reply:
OK, I will undo this change and leave this.